deploy/kubernetes/examples/demo-secret-configuration-file.json

```diff
@@ -0,0 +1,24 @@
+{
+  "demo-system-id1": {
+    "username": "demo-username",
+    "password": "demo-password",
+    "management_address": "demo-management-address",
+    "supported_topologies": [
+      {
+        "topology.block.csi.ibm.com/zone": "demo-zone",
+        "topology.block.csi.ibm.com/dev": "demo-dev"
+      }
+    ]
+  },
+  "demo-system-id2": {
+    "username": "demo-username",
+    "password": "demo-password",
+    "management_address": "demo-management-address",

```

Comment:
consider adding "2" suffix to the second system values

Reply:
done